.github/workflows/main.yml

```diff
@@ -292,24 +290,33 @@ jobs:
   test-node-wrapper:
     needs: workflow-setup
     runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        node-version: [12.x, 18.x]
```

Comment:
What is the issue with node wrapper unit tests on node 19? The integration tests seem to run fine on node 19.

Reply:
Had some issues .... I'll try to add it again